src/Flex/Flex.tsx

```diff
@@ -27,7 +27,7 @@ if (typeof window !== "undefined") {
     !!window?.CSS?.supports?.("gap", "0px") && !(isSafari || isOpera);
 }
 
-export interface FlexProps extends React.HTMLAttributes<any> {
+export interface FlexProps extends Omit<React.AllHTMLAttributes<any>, "as"> {
```

Comment:
👍

Reply:
Will this prevent us from using the prop directly, though?

Reply:
You mean the `as` prop? No, because we have it `FlexProps` but with better type.